pyproject.toml

```diff
@@ -54,11 +54,40 @@ pytest = "*"
 pytest-cov = "*"
 tomli = "*"
 
-[tool.isort]
-profile = "black"
+[tool.ruff]
+target-version = "py310"
```

Comment:
**issue:** support for Python 3.9

this repo is supporting python = "^3.9" (see line 39)
```suggestion
target-version = "py39"
```